### PR TITLE
Upgrade paersonal_information_page module.

### PR DESCRIPTION
**Pull request by richard-willdooit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,6 +45,11 @@ coverage.xml
 # Pycharm
```
Comment (elicoidal) [inline .gitignore]: Please remove this file

File: README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Available addons
 addon | version | summary
```
Comment (elicoidal) [inline README.md:20]: Please format the README file following https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md

Comment (richard-willdooit) [inline README.md:20]: I can't see anything in there about formatting README files?

Also, given that this is a repository wide file, not module specific, I am a bit loathe to making changes beyond what is relevant to what I have changed.

Comment (elicoidal) [inline README.md:20]: There is one README for the repo but you should have one for the module 
as well.

Comment (richard-willdooit) [inline README.md:20]: There is already one in the module, as well.

Comment (elicoidal) [inline README.md:20]: This is the one that should be normalized following OCA template

Comment (pedrobaeza) [inline README.md:20]: You don't even need to change this one, because it's automatically updated with an script.

**Pull request closed without merging**